freshdesk_api.py

```diff
@@ -209,3 +214,30 @@ def api_endpoint(self, id):
 
     def create(self, id, **kwargs):
         return self.client.req(requests.post, self.api_endpoint(id), self.wrapper_name, **kwargs)
+
+class FreshDeskTickets(FreshDeskObjects):
+    api_name = 'ticket'
+    wrapper_name = 'helpdesk_ticket'
+
+    def convert_args_from_dic(self, args):
```

Comment:
You don't want to do that by hand, see [urllib.parse.urljoin](http://devdocs.io/python/library/urllib.parse#urllib.parse.urljoin).

Anyway, I don't think you need that function (see my next comment).